libs/sdk-ui-tests-e2e/cypress/tools/chart.ts

```diff
@@ -131,4 +134,28 @@ export class Chart {
             this.getElement().find(DATA_LABELS_SELECTOR).eq(i).contains(labels[i]).should("exist");
         }
     }
+
+    public assertShortenMetricNameInLegend() {
+        this.assertShortenMetricName(LEGEND_NAME_SELECTOR, 200);
+    }
+
+    public hoverOnChart1stPointInHigh() {
+        cy.get(HIGHCHART_POINT_SELECTOR).first().as("firstElement");
+        cy.get("@firstElement").trigger("mouseover", { force: true });
+    }
+
+    public assertShortenMetricNameInTooltip() {
+        this.assertShortenMetricName(TOOLTIP_TITLE_SELECTOR, 320);
+    }
+
+    public assertShortenMetricNameInHeaderTable() {
+        this.assertShortenMetricName(TABLE_HEADER_TITLE, 590);
+    }
+
+    public assertShortenMetricName(element: string, maxWidth: number) {
+        cy.get(element).invoke("css", "text-overflow").should("equal", "ellipsis");
+        cy.get(element).invoke("width")
+            .should("be.lessThan", maxWidth);
```

Comment:
use equal instead of lessThan, because maxWidth is a number you come up with yourself, it won't make any sense